Formula/helmfile.rb

```diff
@@ -34,7 +36,7 @@ def install
     - name: test
     EOS
     system Formula["kubernetes-helm"].opt_bin/"helm", "init", "--client-only"
-    output = '"stable" has been added to your repositories'
+    output = ""
```

Comment:
Match with empty string is not a good test at all, since it always passes I think.

Reply:
yeah, you are right, I did realize that 😆

Reply:
I will fix it.

Reply:
It looks like there is no output for this line, `shell_output("#{bin}/helmfile -f helmfile.yaml repos")`

Reply:
then a new test is needed and we must validate that the new behavior (having no output) is actually valid, and not indicative of a problem.